pages/search/[id].js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
```

Comment:
I would suggest renaming this file to `dogs/[id].js` or `breeds/[id].js`. This creates more common URLs such as:

- http://localhost:3000/dogs/21
- http://localhost:3000/breeds/21

Reply:
How it is right now (with the filename `pages/search/[id].js`), creates URLs like the following, which are somewhat unusual:

- http://localhost:3000/search/21